### ISSUE TITLE
Unable to deploy suricata with aws cloudformation

### ISSUE THREAD

**Issue by jhimansh** [aws-quickstart/quickstart-amazon-vpc-traffic-mirroring#13]
Hello,
I am trying to deploy suricata with aws cloudformation. I created the stack as mentioned here: https://aws-quickstart.github.io/quickstart-amazon-vpc-traffic-mirroring/
The stack was successfully created and I could see instances created. I added a mirror target and source. All the requirements were fulfilled. 
However, when I connect to the suricata ec2 instance(TargetInstanceASG) and check for suricata status, this is the response I get:
$ systemctl status suricata
Unit suricata.service could not be found.

Even though the stack is successfully created, I am unable to understand why suricata is not installed on the instance. Is there an issue with suricata yaml? Or is there  a support issue with amazon linux 2? Please let me know.
Help will be appreciated. Thanks!

**Comment by ajw4sk** [comment]
Yeah, I had the same issue.  Looking through the template, I'm not seeing anything about installing Suricata either.  What's the story?
